src/twisted/test/test_shortcut.py

```diff
@@ -15,6 +15,9 @@
 except ImportError:
     skipReason = "Only runs on Windows with win32com"
 
+if sys.version_info[0:2] >= (3, 7):
```

Comment:
Doesn't the full comparison (without `[0:2]`) work fine?  Also, I would have expected we would want to fix the test, not skip it?  Or is there some reason this shouldn't be supported.